code/obj/critter/misc.dm

```diff
@@ -1491,7 +1505,7 @@
 
 	CritterDeath()
 		..()
-		src.reagents.add_reagent("viper_venom", 40, null)
+		src.reagents.add_reagent("viper_venom", 100, null)
```

Comment:
Seems unnecessary given how dangerous viper venom is.

Reply:
alright